File: mapper/src/main/java/com/itransition/fanfictionbackend/mapper/user/UserPreviewDtoMapper.java
package com.itransition.fanfictionbackend.mapper.user;

import com.itransition.fanfictionbackend.dto.user.UserPreviewDto;
import com.itransition.fanfictionbackend.mapper.OneWayMapper;
import com.itransition.fanfictionbackend.model.User;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface UserPreviewDtoMapper extends OneWayMapper<User, UserPreviewDto> {}

File: src/main/kotlin/com/itransition/fanfictionbackend/mapper/user/UserPreviewDtoMapper.kt
package com.itransition.fanfictionbackend.mapper.user

import com.itransition.fanfictionbackend.dto.user.UserPreviewDto
import com.itransition.fanfictionbackend.mapper.ToDtoMapper
import com.itransition.fanfictionbackend.model.User
import org.mapstruct.Mapper

@Mapper(componentModel = "spring")
interface UserPreviewDtoMapper : ToDtoMapper<User, UserPreviewDto>
